src/overlords/mining/miner.ts

```diff
@@ -97,6 +110,52 @@ export class MiningOverlord extends Overlord {
 		}
 	}
 
+	checkIfDoubleMine() {
+		if (Game.rooms[this.pos.roomName]) {
+			// console.log('Double mining check');
+			this.source = _.first(this.pos.lookFor(LOOK_SOURCES));
+			const nearby = this.source.pos.findInRange(FIND_SOURCES, 2).filter(source => this.source != source);
+			if (nearby.length > 0) {
+				this.secondSource = nearby[0];
+				// If its over 1 spot away, is there spot in between to mine?
+				if (!this.source.pos.isNearTo(this.secondSource)) {
+					const miningPos = this.source.pos.getPositionAtDirection(this.source.pos.getDirectionTo(this.secondSource.pos));
+					if (!miningPos.isWalkable()) {
+						// console.log(`Double mining found but there is no spot between ${this.secondSource}
+						// ${this.secondSource.pos.print} isWalkable ${miningPos}`);
+						return false;
+					}
+				}
+				if (this.source.id > this.secondSource.id) {
+					// console.log(`This is a disabled mining ${this.directive.name} via source id`);
+					this.isDisabled = true;
+				}
+				return true;
+			}
+		}
+		return false;
+	}
+
+	checkForNearbyMines() {
```

Comment:
looks like this method isn't needed since it's duplicate of `checkIfDoubleMine`... remove if this is the case

Reply:
This one was intended to be for large miners who traverse between two nearby sources vs a stationary miner with 2 sources and 1 spot. But never implemented so I'll remove the dead code